shane_fisher/fundamentals/basic13.rb

```diff
@@ -0,0 +1,120 @@
+# print 1-255
+
+(1..255).each {|num| puts num}
+
+# print odd between 1-255
+
+(1..255).each { |num| puts num if num.odd?}
+
+# print Sum
+
+puts (1..255).reduce(:+)
+
+# print Sum with output
+
+sum = 0
+(0..255).each { |num| puts "New number: " + num.to_s + " Sum: #{sum + num}"; sum += num }
```

Comment:
for lines like this in the future. You could use string interpolation. ex: "New number: #{num} Sum: #{sum + num}"